src/software/ai/hl/stp/tactic/shadow_freekicker_tactic.h

```diff
@@ -14,7 +14,7 @@ class ShadowFreekickerTactic : public Tactic
      * This enum indicates the side the robot running this tactic should shadow on. "Left"
      * and "Right" are from the POV of a robot in the friendly goal looking at the enemy
      * taking the free kick. For example, the tactic using the LEFT enum would shadow
-     * slightly to the left of the vector from the enemy freekicker to the friendly goal
+     * slightly to the left of the vector from the enemy free_kicker to the friendly goal
```

Comment:
where there are comments like here, we should use a space
```suggestion
     * slightly to the left of the vector from the enemy free kicker to the friendly goal
```